engine/enginefuzz_test.go

```diff
@@ -298,3 +298,248 @@ func validateTestCases(t *testing.T, cases []*testCase) {
 		t.Fatalf("failed %d test cases", failures)
 	}
 }
+
+func FuzzNativeHistogramOnInstantQuery(f *testing.F) {
```

Comment:
Maybe we can add `f.Skip()` for now because these tests unearthed compatibility that will take some time to fix?